build/buildall

```diff
@@ -137,6 +139,11 @@ case "$1" in
   FINAL_OP="install"
   ;;
 
+--docgen)
+  FINAL_OP="verify"
+  DOCGEN=true
+  ;;
+
```

Comment:
If `docgen` is not enabled by default, then the Contributing documentation needs to mention how to generate the docs. I guess it is kind of not clear that the optional flag docgen changes the mvn target and that it overlaps with other targets.